cfy_manager/components/prometheus/prometheus.py

```diff
@@ -619,6 +619,10 @@ def _deploy_exporters_configuration():
 
 
 def _prometheus_additional_configuration():
+    # The prometheus_query_lookback_delta is used to render Prometheus'
+    # `query.lookback-delta` flag.  The flag defines the maximum lookback
+    # duration for retrieving metrics during expression evaluations and
+    # federation.
```

Comment:
Might be good to add to the comment why we're adding this- e.g.
"This causes the cluster status to become accurate more quickly when a node is lost" (or whatever it does)